composer.json

```diff
@@ -9,18 +9,19 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^8.3",
         "php-http/httplug": "^1.0 || ^2.0",
         "php-http/client-implementation": "^1.0",
         "php-http/discovery": "^1.7",
-        "psr/log": "~1.0"
+        "psr/log": "^1.0 || ^2.0 || ^3.0"
     },
     "require-dev": {
         "phpunit/phpunit": "^8.4",
         "nyholm/nsa": "^1.1",
         "php-http/curl-client": "^1.0 || ^2.0",
         "php-http/message": "^1.8",
-        "nyholm/psr7": "^1.2"
+        "nyholm/psr7": "^1.2",
+        "phpstan/phpstan": "^2.1"
```

Comment:
Why do we add PHPStan as a direct dependency here?

Reply:
IIRC you can install PHPStan globally locally and use it, right? I mean, it doesn't have to be a direct dependency? Correct me if I'm wrong but lately using SA tools as direct dependencies is not the best practice anymore?